docs/docs/resources/azure-ad/AADApplication.md

```diff
@@ -8,6 +8,7 @@
 | **ObjectId** | Write | String | ObjectID of the app. | |
 | **AppId** | Write | String | AppId for the app. | |
 | **AvailableToOtherTenants** | Write | Boolean | Indicates whether this application is available in other tenants. | |
+| **Description** | Write | String | A free text field to provide a description of the application object to end users. The maximum allowed size is 1024 characters. | |
```

Comment:
We are generating the documentation during release. You can leave it for now, but next time you can save you the trouble 😉